kolibri/plugins/learn/assets/src/views/AssessmentWrapper/index.vue

```diff
@@ -442,11 +442,8 @@ oriented data synchronization.
       setItemId() {
         const index = this.totalattempts % this.assessmentIds.length;
         if (this.randomize) {
-          if (this.userid) {
-            this.itemId = seededShuffle.shuffle(this.assessmentIds, this.userid, true)[index];
-          } else {
-            this.itemId = seededShuffle.shuffle(this.assessmentIds, Date.now(), true)[index];
-          }
+          const seed = this.userid ? this.userid : Date.now();
+          this.itemId = shuffle(this.assessmentIds.slice(0), seed)[index];
```

Comment:
Does the shuffle algorithm shuffle Arrays in place? Might be good to wrap that behaviour if we use it frequently.